code-challenges.test.js

```diff
@@ -16,14 +16,51 @@
 
 const fibonacciLength1 = 6
 // Expected output: [1, 1, 2, 3, 5, 8]
-
 const fibonacciLength2 = 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
+describe("figFunct", () => {
+  it("takes in a number (greater than 2)", () => {
+    expect(figFunct(6)).toEqual([1, 1, 2, 3, 5, 8]);
+  })
+})
+describe("figFunct", () => {
+  it("takes in a number (greater than 2)", () => {
+    expect(figFunct(10)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55]);
+  })
+})
```

Comment:
Remember we only want 1 test per function. So you only want 1 describe and 1 it statement, then we can have multiple expect statements.